README.md

```diff
@@ -0,0 +1,9 @@
+Sublime LSP plugin for Grammarly
+=
+
+[Grammarly](https://grammarly.com/) support for [Sublime's LSP plugin](https://github.com/sublimelsp/LSP), using the [Grammarly language server](https://github.com/znck/grammarly)
+
+Configuration
+==
+
+Make sure you have a recent `node` installed (17.0.0+ recommended) and accessible in sublime's path, even if sublime does not source your `~/.bashrc` (or equivalent).
```

Comment:
I would also suggest adding something about the fact that this server will send the contents of the files to remove server. Maybe something like:

> Privacy warning
> == 
> Note that the contents of the files on which this server is running (Markdown and Tex) will be sent to Grammarly servers and will be subject to Grammarly's [Privacy Policy](https://www.grammarly.com/privacy-policy).
```

Reply:
done